Makefile

```diff
@@ -1579,16 +1578,14 @@ endif
 .PHONY: protos-up-to-date/host
 protos-up-to-date/host: must-start-clean/host grpc/host
 	@if ! git diff --quiet; then \
-		echo 'Please run make grpc.'; \
-		git diff; \
+		./build.assets/tooling/cmd/please-run.sh "protos gRPC" "make grpc"; \
 		exit 1; \
 	fi
 
 .PHONY: must-start-clean/host
 must-start-clean/host:
 	@if ! git diff --quiet; then \
-		echo 'This must be run from a repo with no unstaged commits.'; \
-		git diff; \
+		./build.assets/tooling/cmd/please-run.sh "protos gRPC" "make grpc"; \
```

Comment:
This looks wrong? I don't think must-start-clean has a particular target it wants to run.